evm/p2p/auth.py

```diff
@@ -170,18 +182,18 @@ def create_auth_ack_message(self, nonce):
             msg = self.ephemeral_pubkey.to_bytes() + nonce + token_flag
         return msg
 
-    def encrypt_auth_ack_message(self, ack_message):
+    def encrypt_auth_ack_message(self, ack_message: bytes) -> bytes:
         if self.got_eip8_auth:
             # The EIP-8 version has an authenticated length prefix.
-            prefix = struct.pack('>H', len(ack_message) + ecies.encrypt_overhead_length)
+            prefix = struct.pack('>H', len(ack_message) + ENCRYPT_OVERHEAD_LENGTH)
```

Comment:
This was the first bug caught by mypy. It didn't show up when running the light client because by default we don't use the EIP8 handshake

Reply:
That's subtly really awesome